addons/website_sale/views/templates.xml

```diff
@@ -1879,6 +1879,27 @@
                         aria-label="Edit this address">
                         <i class="fa fa-pencil me-1"/>Edit
                     </a>
+                    <t t-set="archive_address" t-value="'/shop/address/archive/address'"/>
```

Comment:
The buttons are missing icons and are not properly aligned in flex or grid layout. What I suggest use a grid layout: place two buttons in the first row and the third button in the second row since they won't fit in a single row.